signup.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>SpaceWalkers Sign Up</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
+  <link href="css/index.css" rel="stylesheet">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+ <!-- navigation code here -->
+  <div class='navigation'>
+    <h1><a href='index.html'>SpaceWalkers</a></h1>
+    <nav>
+      <a href='index.html'>Home</a>
+      <a href='#'>About Us</a>
+      <a href='#'>Blog</a>
+      <a href='#'>Contact</a>
+      <a href='#'>Sign In</a>
+    </nav>
+  </div>
+  <div class="container home">
+
+    <header class="form-intro">
+      <!-- image code here -->
+
+       <img class='banner-img' src='img/main-header.png'>
+
+      <h2>Come Fly Away!</h2>
+    
+      <p>So, you want to join our newsletter? No Problem! We just need to know a little bit more about you first. Please take a moment to fill out our survey, and a customer representative will get back to you in 1-2 business days.</p>
+    </header>
+    
+
+    <div class='form-sheet'>
```

Comment:
Good job with the form. Generally speeking we don't use id's anymore. Nothing against using them. Its just more common to use name instead. Once you get to react you will set onChange handlers on each of the inputs. Then in the onChange function. You can get the name of the input that is being changed by accessing e.targert.name;